src/Microsoft.AspNet.Mvc.Core/Filters/FormatFilter.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using Microsoft.Framework.OptionsModel;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Microsoft.AspNet.Mvc.Core.Filters
+{
+    public class FormatFilter : IFormatFilter
+    {
+        public void OnResourceExecuting([NotNull] ResourceExecutingContext context)
+        {
+            var options = (IOptions<MvcOptions>)context.HttpContext.RequestServices.GetService(
+                typeof(IOptions<MvcOptions>));
```

Comment:
use `GetRequiredService<T>()` you may need to add `using Microsoft.Framework.DependencyInjection`